domains/mikkel.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "MIkkelMadsen",
+           "email": "mikkelbm@live.dk",
+           "discord": "222777970950012928"
+        },
+    
+        "record": {
+            "A": ["192.165.250.202"]
```

Comment:
"This website is not available. If you are trying to visit your own website, contact your hosting provider for assistance."